staging/src/k8s.io/apimachinery/pkg/util/portforward/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package portforward
+
+const (
+	PortForwardV1Name   = "portforward.k8s.io"
+	SpdyTunnelingPrefix = "SPDY/3.1+"
+	TunnelPortForwardV1 = SpdyTunnelingPrefix + PortForwardV1Name
```

Comment:
nit, make it clear this is for use with websockets:

```suggestion
	WebsocketSPDYTunnelingPrefix = "SPDY/3.1+"
	WebsocketSPDYTunnelingPortForwardV1 = WebsocketSPDYTunnelingPrefix + PortForwardV1
```